apps/events/views.py

```diff
@@ -65,11 +69,29 @@ def get_success_url(self):
     def post(self, request, *args, **kwargs):
         self.object = self.get_object()
         form = self.get_form()
+
         if form.is_valid():
             if form.instance.type == "private":
                 form.instance.is_visible = False
             else:
                 form.instance.is_visible = True
+
+            # Check if a new image for the event is uploaded
+            new_image_uploaded = "image" in request.FILES and request.FILES[
+                "image"
+            ] != form.initial.get("image")
```

Comment:
One-line constructions are hard to read
Please use a simple if

You can use somethink like
```
if request.FILES.get("image", "") != form.initial.get("image"):
     new_image_uploaded = True
```